pkgs/by-name/he/headlessDisplayHook/package.nix

```diff
@@ -0,0 +1,17 @@
+{
+  lib,
+  stdenv,
+  makeSetupHook,
+  xvfb,
+  mesa,
+}:
+if stdenv.hostPlatform.isLinux then
```

Comment:
Please just use xvfb-run.

Reply:
Yes, and it'll be around for all the subsequent phases, which we don't want. Just run the tests in `xvfb-run`.